holoviews/plotting/bokeh/element.py

```diff
@@ -1251,10 +1252,19 @@ def _init_glyph(self, plot, mapping, properties):
             del properties['legend_label']
 
         # ALERT: This only handles XYGlyph types right now
-        if 'x' in mapping and mapping['x'] in plot.extra_x_ranges:
-            properties['x_range_name'] = mapping['x']
-        if 'y' in mapping and mapping['y'] in plot.extra_y_ranges:
-            properties['y_range_name'] = mapping['y']
+        if 'x' in mapping:
+            x = mapping['x']
+            if isinstance(x, Field):
```

Comment:
This was added in Bokeh 3. There is in the utility folder the following function `property_to_dict`